content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,150 @@
+
+{{< alert icon="💡" context="info" text="SDP (Session Description Protocol) is a protocol that is used to describe multimedia sessions. The SDP answer is a message sent by a server in response to an SDP offer from a client. The offer and answer are used to establish a session between a client and server, allowing them to exchange media." />}}
+
+Once the browser sets the SDP offer and answer, it will send STUN requests to
+the server. The browser and server then execute the DTLS handshake as part of the standard
+WebRTC connection establishment. It is similar to the TLS handshake described
+[here](../secure-comm/tls##comparing-tls-1.3-to-tls-1.2), with the differences being that DTLS is
+datagram-based and how the server generates session keys.
+
+1. The browser sends a `ClientHello` message to advertise its supported cipher suites
+   and other protocol options to the server.
+2. The server responds with a `ServerHello` message to indicate which cipher suite
+   and other options it has selected for communication.
+3. The server sends its certificate along with a `ServerKeyExchange` message to the client
+   that includes the necessary information for the client to generate a premaster secret.
+4. The browser verifies the server's certificate and sends a `ClientKeyExchange` message,
+   which contains the premaster secret encrypted using the server's public key.
+5. The server decrypts the premaster secret and uses it, along with its private key, to
+   generate the session keys.
+
+A successful DTLS handshake only provides confidentiality and integrity. Authenticity is
+achieved by succeeding a [Noise handshake](../secure-comm/noise) following
+the DTLS handshake. Messages on each `RTCDataChannel` are framed using a message-framing
+mechanism described
+[here](https://github.com/libp2p/specs/blob/master/webrtc/README.md#multiplexing).
+
+<!-- TO ADD: DIAGRAMS ONCE READY + CONTEXT -->
+
+### Coming soon: Browser-to-Browser
+
+Eventually, libp2p will have support for two-way communication between two
+browsers in real-time.
+
+The technical specification and initial implementations of WebRTC
+Browser-to-Browser connectivity is planned for release in early 2023.
+Track the progress [here](https://github.com/libp2p/specs/issues/475).
+
+{{< alert icon="💡" context="note" text="See the WebRTC <a class=\"text-muted\" href=\"https://github.com/libp2p/specs/blob/master/webrtc/README.md\">technical specification</a> for more details." />}}
+
+## Comparing WebRTC and WebTransport
+
+While WebRTC and WebTransport are both web-based approaches that enable real-time
+communication between nodes, there are key differences. WebRTC supports peer-to-peer
+connections, while WebTransport only supports client-server connections.
+
+The underlying protocols in WebRTC and WebTransport are different, although both
+protocols share many of the same properties. WebTransport is also an
+alternative to the data channels available in WebRTC.
```

Comment:
I don't think the first sentence adds much information. I don't understand the second sentence.